CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * Add support for subclassing using `Class.new` to `Lint/InheritException`. ([@houli][])
 * [#6779](https://github.com/rubocop-hq/rubocop/issues/6779): Add new cop `Style/NegativeUnless` that checks for unless with negative condition. ([@tejasbubane][])
 * [#6649](https://github.com/rubocop-hq/rubocop/pull/6649): Layout/AlignHash supports list of options. ([@stoivo][])
-
+* Respect DisabledByDefault for custom cops. ([@XrXr][])
```

Comment:
This should go in the `master` section of the changelog, under `Bug fixes`.

Reply:
Updated.